test/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ file(TO_CMAKE_PATH "${PROJECT_SOURCE_DIR}" CELERITAS_SOURCE_DIR)
 configure_file(celeritas_test_config.h.in celeritas_test_config.h @ONLY)
 
 set(_harness_sources
+  Histogram.cc
```

Comment:
Also since we now have test/corecel/random, should this class live there since it's used for random testing?